classes/file/TemporaryFileManager.inc.php

```diff
@@ -57,6 +57,16 @@ function deleteFile($fileId, $userId) {
 	}
 
 	/**
+	 * Delete a file by path.
+	 * For temporary files that are not in the DB, e.g. exported xml files.
+	 * @param $filePath string the location of the file to be deleted
+	 * @return boolean returns true if successful
+	 */
+	function deleteFileByPath($filePath) {
```

Comment:
(I don't love the TemporaryFileManager in general, and I'm now directing your work without much recent experience with it -- so if you think we're going down a rabbit hole, please stop me :)